docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -246,13 +270,20 @@ public class GreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public String hello() {
-        return "hello\n";
+        return "hello";
     }
 }
 ----
 
 If you stopped the application, restart the application with `./mvnw compile quarkus:dev`.
-Then check that http://localhost:8080/hello/greeting/quarkus returns `hello quarkus`.
+Then check that the endpoint returns `hello quarkus` as expected:
+
+```
```

Comment:
This is Markdown, please use a `[source]` block.

Reply:
whoops - sorry, bad habit :) fixed.

Reply:
fixed in latest commit.